airflow/www/app.py

```diff
@@ -70,7 +72,20 @@
 
 def create_app(config=None, testing=False):
     """Create a new instance of Airflow WWW app."""
-    flask_app = Flask(__name__)
+    connexion_app = connexion.FlaskApp(__name__)
+
+    @connexion_app.app.before_request
```

Comment:
Here the question is if that exemption here from CSRF is teh right one 

TODO: also add FAB cc: @vincbeck  ?

Reply:
Correct. The path is `/auth/fab/v1`. Ideally we should have a set containing all the paths that need to ignore CSRF. SO far I can see only 2: Airflow API and Fab API

Reply:
Yes. If we go that direction we should likely propose a better way than it is currently (it's a bit messy).